docs/tuning_fledge.rst

```diff
@@ -213,7 +213,25 @@ The tuning of the south service allows the way the buffering is used within the
 
-Tuning between performance, latency and memory usage is always a balancing act, there are situations where the performance requirements mean that a high latency will need to be incurred in order to make the most efficient use of the communications between the micro services and the transnational performance of the storage engine. Likewise the memory resources available for buffering may restrict the performance obtainable.
+Tuning between performance, latency and memory usage is always a balancing act, there are situations where the performance requirements mean that a high latency will need to be incurred in order to make the most efficient use of the communications between the micro services and the transactional performance of the storage engine. Likewise the memory resources available for buffering may restrict the performance obtainable.
+
+Reading Latency
+---------------
+
+Closely related to buffer usage is reading latency in the south service. This is a measure of the delay between the south service receiving a new reading and that reading appearing in the storage subsystem. We deliberately delay the forwarding of readings from the south service to storage in order to create blocks of multiple readings to send per call to the storage layer. This increases the overall throughput of the south to storage interface at the cost of increasing the latency. There are two settings that come into play when defining this, the maximum latency we will accept and the maximum number of readings we will buffer.
+
+In situations where readings are arriving in the south service relatively frequently these can be set to values to allow data to build up reasonable size blocks of readings to send and hence be more efficient in sending the data to the storage layer. However if data does not arrive frequently or is not predictable in the way it arrives then these settings may cause unexpected latency and delays within the system.
+
+The buffering subsystem within the south service will buffer readings in the south as they arrive. It checks the time difference between the oldest buffered reading and the current time to see if the maximum latency setting is about to be exceeded. If it is it will send the buffered data. If it is not then it will check the number of readings buffered and if that is about to exceed the maximum allowed number of buffered readings it will then send all the buffered readings. No further checks are done until the next reading arrives.
+
+Therefore, if readings do not arrive very frequently, or the south plugin is asynchronous and data arrives sporadically, then it may not check the buffer status for more than the maximum configured latency period. This results in that maximum latency being exceeded. When this occurs a warning message will be logged in the system logs.
```

Comment:
> This results in that maximum latency being exceeded. When this occurs a warning message will be logged in the system logs.

As a result, this can lead to the maximum latency being exceeded. When this happens, a warning message will be logged in the system logs.